src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -176,10 +177,9 @@ class Comment extends React.Component<Props, State> {
                             <ActivityTimestamp date={createdAtTimestamp} />
                         </div>
                         {isEditing ? (
-                            /* NOTE: Inline editing is not currently supported for comments */
                             <ApprovalCommentForm
                                 isDisabled={isDisabled}
-                                className={classNames('bcs-activity-feed-comment-input', {
+                                className={classNames('bcs-activity-feed-comment-input', 'bcs-Comment-editor', {
```

Comment:
This may need a second look after the recent changes to `CommentForm`.

Reply:
Probably, can rebase and check